examples/all.py

```diff
@@ -171,6 +171,8 @@ def run_example(example, reporter=None):
         else:
             mod.main()
         return True
+    except KeyboardInterrupt:
+        sys.exit(0)
```

Comment:
But I don't think it should exit(0) either. Interrupting the program should give a negative error code.